features/step_definitions/networking.rb

```diff
@@ -989,3 +989,31 @@
   raise "OVN is not running correctly! Check one of your ovnkube-node pod" unless desired_ovnkube_node_replicas == available_ovnkube_node_replicas && available_ovnkube_node_replicas != 0
   raise "OVN is not running correctly! Check one of your ovnkube-master pod" unless desired_ovnkube_master_replicas == available_ovnkube_master_replicas && available_ovnkube_master_replicas != 0
 end
+
+Given /^I store multicast keyval according to the networktype in the#{OPT_SYM} clipboard$/ do |cb_keyval|
+  ensure_admin_tagged
+  _admin = admin
+  cb_name ||= "keyval_value"
+  @result = _admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
+  raise "Unable to find corresponding networkType pod name" unless @result[:success]
+  if @result[:response] == "OpenShiftSDN"
+     cb[cb_keyval] = "netnamespace.network.openshift.io/multicast-enabled=true"
+  else
+     cb[cb_keyval] = "k8s.ovn.org/multicast-enabled=true"
+  end  
+  logger.info "The corresponding networktype multicast keyval is stored in the #{cb_keyval} clipboard."
+end
+
+Given /^I store multicast namespace according to the networktype in the#{OPT_SYM} clipboard$/ do |cb_namespace|
```

Comment:
this step is a copy of the one above. intentional? Or I'm missing something?

Reply:
@akostadinov  create Given /^I enable multicast for the "(.+?)" namespace now.
New log from testing ovn cluster:
https://privatebin-it-iso.int.open.paas.redhat.com/?b29383740e18b36d#c2jFDsUgg39zYNyBJaRADG8YUYYCtZcYDXSuRJIPnTI=

New log from testing sdn cluster:
https://privatebin-it-iso.int.open.paas.redhat.com/?538718d77b3cbbdc#23kXoYOHTbWMQ2LkQyzMROjkJFq/uxcJ754Px7qZf30=